go.mod

```diff
@@ -3,25 +3,25 @@ module sigs.k8s.io/controller-runtime
 go 1.16
 
 require (
-	github.com/evanphx/json-patch v5.6.0+incompatible
+	github.com/evanphx/json-patch v4.12.0+incompatible
```

Comment:
This is now sync with k/k (note: code should be the same as the previous tag as the real v5 is under the /v5 import path)